src/openrct2/scenario/Scenario.cpp

```diff
@@ -608,9 +608,6 @@ bool scenario_prepare_for_save()
     if (gScenarioObjective.Type == OBJECTIVE_GUESTS_AND_RATING)
         gParkFlags |= PARK_FLAGS_PARK_OPEN;
 
-    // Fix #2385: saved scenarios did not initialise temperatures to selected climate
-    climate_reset(gClimate);
```

Comment:
This undoes [this fix](https://github.com/OpenRCT2/OpenRCT2/commit/9e203998931b6b5285d4ee43bd963f822c9dc8f5). Isn't it necessary anymore?

Reply:
I have tested it by opening several scenarios with different climates, and setting it to hyperspeed. As far as I can tell the weather changes appropriately.